### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by devnote-dev** [devnote-dev/yagpdb-ccs#9] [Ideas]
**Welcome!**
This discussion is more for ideas and suggestions if you don't want to PR or want to post here for whatever reason, I don't mind. :P

**Comment by pazurkota** [comment]
I suggest to have ability to send attachments (photos, screenshots) in report system :)

**Reply by devnote-dev** [reply]
that seems doable, I'll check for attachment &/or URL compatibility and go from there. Thx for the suggestion! 👍

**Reply by pazurkota** [reply]
nice

**Reply by pazurkota** [reply]
that would be very helpful 👍
